assets/scss/modules/_phase-banner.scss

```diff
@@ -0,0 +1,8 @@
+.header-banner-wrapper {
+    max-width: 960px;
+    margin: 0 auto;    
+    display: block;
+    @media screen and (max-width: 986px) {
+        padding: 0px 15px 0px;
```

Comment:
@roundrobin99 Can you put that screenshot in the PR description, not in line comments.